docs/sdks/uid2-sdk-ref-android.md

```diff
@@ -131,15 +185,20 @@ This section includes the functions and variables that are part of the UID2Manag
 ### Functions
 
 The following functions are available as part of the UID2Manager API:
+- [generateIdentity()](#generateidentity)
 - [setIdentity()](#setidentity)
 - [resetIdentity()](#resetidentity)
 - [refreshIdentity()](#refreshidentity)
 - [getAdvertisingToken()](#getadvertisingtoken)
 - [setAutomaticRefreshEnabled()](#setautomaticrefreshenabled)
 
+#### generateIdentity()
+
+Generate a UID2 Identity using <Link href="../ref-info/glossary-uid#gl-dii">Directly identifying information (DII)</Link>. For instructions, see [Configure the UID2 mobile SDK](../guides/integration-mobile-client-side.md#configure-the-uid2-mobile-sdk) in the *Client-Side Integration Guide for Mobile*.
+
 #### setIdentity()
 
-Sets the UID2 Identity to be managed by the SDK.
+Sets a UID2 Identity, created server-side, to be managed by the SDK. For details, see [Configure the UID2 Mobile SDK for Your Mobile App](../guides/integration-mobile-client-server.md#configure-the-uid2-mobile-sdk-for-your-mobile-app) in the *Client-Server Integration Guide for Mobile*.
```

Comment:
Maybe rename _Configure the UID2 Mobile SDK for Your Mobile App_ to _Configure the UID2 mobile SDK_ too to align both integ guide section names

Reply:
Done.